bvs-widgets/vhl_clusters_widget.php

```diff
@@ -65,19 +74,25 @@ function update($new_instance, $old_instance) {
         $instance['title'] = strip_tags($new_instance['title']);
         $instance['url'] = strip_tags($new_instance['url']);
         $instance['results'] = strip_tags($new_instance['results']);
+        $instance['image'] = strip_tags($new_instance['image']);
+        $instance['columns'] = strip_tags($new_instance['columns']);
+        $instance['extra_css'] = strip_tags($new_instance['extra_css']);
 
         if(!empty($instance['url']) and strpos($instance['url'], '?') === false) {
             $instance['url'] = $instance['url'] . "?";
         }
 
-        $this->url = $instance['url'];
-
         $url_dia_ws = file_get_contents($instance['url'] . "&debug=true");
-        $url_dia_ws = explode("<br/><!DOCTYPE", $url_dia_ws);
+        $url_dia_ws = explode("<br/><!DOCTYPE", $url_dia_ws)
```

Comment:
@moacirmoda 

Favor corrigir a ausência de ponto e virgula nesta linha pois ao atualizar o mesmo apresenta no WP:

Parse error: syntax error, unexpected T_VARIABLE in /home/apps/bvs.br/wp-ses-sp/htdocs/wp-content/plugins/bvs-site/bvs-widgets/vhl_clusters_widget.php on line 87